mobile/Ihc/models/Patient.js

```diff
@@ -120,8 +120,6 @@ Patient.schema = {
     birthday: 'string',
     gender: 'int', // 1 = boy, 2 = girl, 0 = undefined
     phone: 'string?',
-    motherHeight: 'double?',
```

Comment:
why did we remove these fields? Did daryn or javi say anything?

Reply:
wait I didn't make these changes....

Reply:
It looks like brent added some commits, so clear that up with him